src/bindist/autodos7.bat

```diff
@@ -0,0 +1,6 @@
+@ECHO Off
+PATH D:\OPENDOS;D:\EMUBIN;D:\
+VERIFY OFF
+PROMPT [OpenDOS] $P$G
+SET OPENDOSCFG=D:\OPENDOS
+CALL D:\EMUBIN\UXMACROS.BAT
```

Comment:
It's not about the result, but only about the fact that the command.com interpreter actually interprets every single line, even empty ones at the end. That's why batch files from dos/windows never have newlines at the end or on a line by themselves.

If you think it's better for readability/versioning to still have them, I'm fine with that. It's just unusual with batch files.